src/core/hle/service/time/interface.h

```diff
@@ -7,10 +7,11 @@
 #include "core/hle/service/time/time.h"
 
 namespace Service::Time {
-
+class SharedMemory;
```

Comment:
There should be newlines on both sides of the forward declaration so things aren't smushed together.

```suggestion
namespace ServiceTime {

class SharedMemory;

class Time final : public Module::Interface {
// ...
```